### DISCUSSION TITLE
Followers didn't transfer while migrating back to the same account I previously migrated away from

### DISCUSSION THREAD

**Discussion by enzocomics** [mastodon/mastodon#20963] [Q&A]
Hi there. I'm wondering if anyone has had a similar experience, or any insight. Here's what I did this past week:

- Created an initial account on mastodon.art
- Initiated the migration from the mastodon.art account to a personal instance account
- Followers were successfully transferred over
- Picked nose for a couple of days
- Initiated the migration from the personal instance account back to the mastodon.art account
- This time, followers were not transferred over
- Migration tool says it completed, and is on cooldown for 30 days, preventing another migration attempt

Unsure if, but may be, related:
- When I initiated the migration from the new account back to original mastodon.art account, the redirect from mastodon.art to the new account was still active (I forgot to turn it off)
- One user who initially followed me on mastodon.art, and then was migrated with my account over to the personal instance, reports that they are now unable to follow my account on mastodon.art (attempted on multiple clients, and they all report `this action is not allowed` or `an error occurred`).

Now, there's a line in the [Account Migration documentation](https://docs.joinmastodon.org/user/moving/) that states:

> Moving your account is the same as redirecting your account, but it will also ***irreversibly*** force everyone to unfollow your current account and follow your new account

Is this what it means when it says ***irreversible***? Thankfully, I'm not *too* attached to either account yet, so it's okay if those followers are stuck or locked out forever. But I *would* love to know what exactly happened.

Thanks for your time!

**Comment by smutek** [comment]
Just here adding a "me too" as I seem to be having the same problem. I just migrated back to my account on an instance I was previously on and lost all of my followers. 😢 

I:
- originally had an account on kolektiva.social
- in January '23 migrated to mstdn.social
- Migration completed successfully and all followers were moved
- Today, June 30 '23, migrated back to the old account on kolektiva.social
- Forgot to turn off redirect on kolektiva before migrating
- mstdn admin screen shows all followers (41) have been migrated
- mstdn profile shows I still have 41 followers
- kolektiva.social shows zero followers

I'm not sure what to do but I'm hoping that if I try again in 30 days maybe it will work then, since the migration did not remove my old followers.

**Reply by viktorix** [reply]
Did followers ever migrate or did you lose them?

**Reply by smutek** [reply]
I didn't try again, I just stayed on my current instance. If I were to try again I have a feeling that turning off the redirect on the target instance before running the migration will do the trick.

**Comment by viktorix** [comment]
Appears I have the same issue.
